utils/maintenance/Contrib_List_Command.php

```diff
@@ -5,6 +5,19 @@
 
 final class Contrib_List_Command {
 
+	/**
+	 * Packages excluded from contributor list generation.
+	 * @var array
```

Comment:
Nitpick: there should a blank comment line above the first `@tag`. (not a blocker as the Docs rules are currently not applied yet anyway)